data/plugins.json

```diff
@@ -157,23 +206,33 @@
     "safari": 10,
     "node": 6,
     "ios": 10,
+    "chrome-flag": null,
+    "edge-flag": 13,
+    "node-flag": 6,
```

Comment:
There's no node release with v8 5.6, which fixes the aforementioned tail call crash.